src/algorithms/ctmrg.jl

```diff
@@ -330,13 +142,13 @@ function ctmrg_iter(state, env::CTMRGEnv{C,T}, alg::CTMRG) where {C,T}
     ϵ = 0.0
 
     for _ in 1:4
-        env, _, _, ϵ₀ = left_move(state, env, alg.projector_alg)
+        env, info = left_move(state, env, alg.projector_alg)
         state = rotate_north(state, EAST)
         env = rotate_north(env, EAST)
-        ϵ = max(ϵ, ϵ₀)
+        ϵ = max(ϵ, info.ϵ)
     end
 
-    return env, ϵ
+    return env, (; ϵ)
```

Comment:
While I like the namedtuples for the information, I don't think we should make the names unicode symbols, as that can be really annoying on certain platforms.

Reply:
Yes, I think you mentioned this before and I forgot. Somehow I like unicode symbols a bit too much...

Reply:
Me too :upside_down_face: